subsys/bluetooth/host/classic/at.c

```diff
@@ -119,24 +119,55 @@ static int get_cmd_value(struct at_client *at, struct net_buf *buf,
 	return 0;
 }
 
-static int get_response_string(struct at_client *at, struct net_buf *buf,
-			       char stop_byte, enum at_state state)
+static bool is_stop_byte(char target, char *stop_string)
+{
+	size_t len;
+
+	len = strlen(stop_string);
+
+	for (size_t i = 0; i < len; i++) {
+		if (stop_string[i] == target) {
+			return true;
+		}
+	}
+	return false;
```

Comment:
Can't all of this be simplified to:
```c
	return (strchr(stop_string, target) != NULL);
```
The function name seems quite strange though, since it's a general purpose character search function.

Reply:
Updated.